cli/fathom_web/commands/pick.py

```diff
@@ -13,14 +12,25 @@
           type=Path(exists=True, file_okay=False, writable=True, dir_okay=True, allow_dash=True))
 @argument('number', type=int)
 def main(from_dir, to_dir, number):
-    """Move a given number of random things from one directory to another.
+    """Move a given number of HTML files and any extracted resources from one directory to another.
 
     Ignore hidden files.
 
     This is useful for dividing a corpus into training, validation, and testing
     parts.
-
     """
-    files = [f for f in listdir(from_dir) if not f.startswith('.')]
+    # Make these strings into ``Path``s so they are easier to work with
+    from_dir = pathlib.Path(from_dir)
+    to_dir = pathlib.Path(to_dir)
+
+    files = [f for f in from_dir.glob('*.html')]
     for file in sample(files, number):
-        move(join(from_dir, file), to_dir)
+        # If the file has resources, we must move those as well
+        if (from_dir / 'resources' / file.stem).exists():
+            # Make sure we don't overwrite an existing resources directory
+            if (to_dir / 'resources' / file.stem).exists():
+                raise RuntimeError(f'Tried to make directory {(to_dir / "resources" / file.stem).as_posix()}, but it'
```

Comment:
It seems to me we should raise click.Abort instead: https://click.palletsprojects.com/en/7.x/exceptions/#which-exceptions-exist. It's a teeny bit more descriptive, and it probably results in a more nicely displayed error.